src/aleph_client/commands/utils.py

```diff
@@ -223,6 +228,64 @@ def __rich_console__(self, console, options):
         yield self.progress
         yield self.table
 
+# This is a copy from aleph-vm
+
+class LoadAverage(BaseModel):
+    load1: float
+    load5: float
+    load15: float
+
+    @classmethod
+    def from_psutil(cls, psutil_loadavg: tuple[float, float, float]):
+        return cls(
+            load1=psutil_loadavg[0],
+            load5=psutil_loadavg[1],
+            load15=psutil_loadavg[2],
+        )
+    
+class CoreFrequencies(BaseModel):
+    min: float
+    max: float
+
+    @classmethod
+    def from_psutil(cls, psutil_freq: psutil._common.scpufreq):
+        min_ = psutil_freq.min or psutil_freq.current
+        max_ = psutil_freq.max or psutil_freq.current
+        return cls(min=min_, max=max_)
```

Comment:
Why does `aleph-client` need to know the CPU load average and frequencies of the user's machine ?